projects/plugins/boost/app/assets/src/js/features/page-cache/meta/meta.tsx

```diff
@@ -1,19 +1,69 @@
-import { Button } from '@automattic/jetpack-components';
-import { __ } from '@wordpress/i18n';
+import { Button, Notice } from '@automattic/jetpack-components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
 import ChevronDown from '$svg/chevron-down';
 import ChevronUp from '$svg/chevron-up';
 import Lightning from '$svg/lightning';
 import styles from './meta.module.scss';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+import { usePageCache } from '$lib/stores/page-cache';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
 
 const Meta = () => {
 	const [ isExpanded, setIsExpanded ] = useState( false );
+	const [ query, mutation ] = usePageCache();
+
+	const settings = query?.data;
+	const setSettings = mutation.mutate;
+
+	const setLogging = ( newValue: boolean ) => {
+		if ( ! setSettings || ! settings ) {
+			return;
+		}
+
+		setSettings( {
+			...settings,
+			logging: newValue,
+		} );
+	};
+
+	const setExceptions = ( newValue: string ) => {
+		if ( ! setSettings || ! settings ) {
+			return;
+		}
+
+		setSettings( {
+			...settings,
+			exceptions: newValue.split( '\n' ).map( item => item.trim() ),
+		} );
+	};
+
+	const totalExceptions = settings?.exceptions.length || 0;
 
 	return (
 		<div className={ styles.wrapper }>
 			<div className={ styles.head }>
 				<div className={ styles.summary }>
-					{ __( 'No exceptions or logging.', 'jetpack-boost' ) }
+					{ totalExceptions === 0 && ! settings?.logging ? (
+						__( 'No exceptions or logging.', 'jetpack-boost' )
+					) : (
+						<>
+							{ totalExceptions > 0 ? (
```

Comment:
We're checking for `totalExceptions` equality twice, that results in both messages like `No Exceptions or logging` and `No exceptions`. Can the logic be changed so that we display more precise messages, like `No exceptions` and `Logging isn't enabled`?

Reply:
I'm not sure what you mean here. I don't see how `No exceptions or logging.` and `No exceptions.` would show up at the same time. Did you actually manage to see this?

Reply:
As discussed over Slack, I'm leaving this for now.